### PR TITLE
Update 6.0.0-rc.1.md with support and servicing details

### PR DESCRIPTION
**Pull request by rbhanda**

### CODE REVIEW

File: release-notes/6.0/preview/6.0.0-rc.1.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ You can check your .NET SDK version by running the following command. The exampl
 $ dotnet --version
 6.0.100-rc.1.21458.32
 ```
+### .NET 6.0 RC1 Support and Servicing
+ 
+.NET 6 RC1 is supported for use in production, which means you can build an app using .NET 6 RC1, deploy this app to a production environment, and expect the level of support and servicing you would normally get for a shipped release such as .NET 5.0. We will continue to support RC1 for 30 days after RC2 ships and support RC2 for 30 days after .NET 6.0 GA ships.
```
Comment (richlander) [inline release-notes/6.0/preview/6.0.0-rc.1.md:41]:
```suggestion
.NET 6 RC1 is supported for use in production, which means you can build an app using .NET 6 RC1, deploy this app to a production environment, and expect the level of support and servicing you would normally get for a stable (non-preview) release such as .NET 5. We will continue to support RC1 for 30 days after RC2 ships and support RC2 for 30 days after .NET 6 GA ships.
```

File: release-notes/6.0/preview/6.0.0-rc.1.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ You can check your .NET SDK version by running the following command. The exampl
...
+ 
+Any bug fixes including security fixes for RC1 and beyond will be handled similar to any other shipped product version like .NET 5.0. These servicing fixes will be made available as a monthly rollup in the next successive RC or GA release i.e. fixes for RC1 will be rolled up into RC2, and fixes for RC2 will be rolled up into .NET 6.0 GA.
```
Comment (richlander) [inline release-notes/6.0/preview/6.0.0-rc.1.md:43]:
```suggestion
.NET 6 RC1 will not be updated after it is released. You will need to update your environment to .NET 6 RC2 and the final .NET 6 stable release for future improvements, including security updates.
```

File: release-notes/6.0/preview/6.0.0-rc.1.md [hunk 3]
```diff
@@ -36,6 +36,12 @@ You can check your .NET SDK version by running the following command. The exampl
 $ dotnet --version
 6.0.100-rc.1.21458.32
 ```
+### .NET 6.0 RC1 Support and Servicing
+ 
+.NET 6 RC1 is supported for use in production, which means you can build an app using .NET 6 RC1, deploy this app to a production environment, and expect the level of support and servicing you would normally get for a stable (non-preview) release such as .NET 5. We will continue to support RC1 for 30 days after RC2 ships and support RC2 for 30 days after .NET 6 GA ships.
```
Comment (richlander) [inline release-notes/6.0/preview/6.0.0-rc.1.md:41]:
```suggestion
.NET 6 RC1 is supported for use in production, which means you can build an app using .NET 6 RC1, deploy this app to a production environment, and expect the level of support and servicing you would normally get for a stable (non-preview) release such as .NET 5. We will continue to support RC1 for 30 days after RC2 ships and support RC2 for 30 days after .NET 6 GA ships.

You can contact the team directly at dotnet@microsoft.com if you run into challenges in your production environment.
```

**Review by jamshedd — APPROVED** [review]

File: release-notes/6.0/preview/6.0.0-rc.1.md (hunk 2 above)
Comment (karelz) [reply release-notes/6.0/preview/6.0.0-rc.1.md]: Building on to of @richlander's suggestion:
```suggestion
.NET 6 RC1 will not be updated after it is released. You will need to update your environment to .NET 6 RC2 and .NET 6 GA for future improvements, including security updates.
```

**Review by karelz — CHANGES_REQUESTED** [review]
We should edit also [release-notes/6.0/README.md](https://github.com/dotnet/core/tree/main/release-notes/6.0) ... at the top it says ".NET 6 is currently in `Preview` and if currently not supported.

We need to modify it to go-live instead, and/or perhaps mention it in the table with listed releases?

**Comment by rbhanda** [comment]
> We should edit also [release-notes/6.0/README.md](https://github.com/dotnet/core/tree/main/release-notes/6.0) ... at the top it says ".NET 6 is currently in `Preview` and if currently not supported.
> 
> We need to modify it to go-live instead, and/or perhaps mention it in the table with listed releases?

I have removed the ".NET 6 is currently in Preview and if currently not supported." from .NET 6.0 readme.md file

**Review by karelz — APPROVED** [review]

**Pull request merged**